### ISSUE TITLE
beforeSendNotification - event is dependent on 'Queue for Notifications' 

### ISSUE THREAD

**Issue by anchovy** [verbb/formie#182]
**Description**
[beforeSendNotification](https://verbb.io/craft-plugins/formie/docs/developers/events#the-beforeSendNotification-event) event works differently depending on the sitting for _Use Queue for Notifications_

If _Use Queue for Notifications_ is disabled you are able to modify the events submission data i.e. change the 'to', 'bcc', subject line etc - but when the craft queue is used these changes are ignored.

**Steps to reproduce**
1. Add an email to a form with the _To_ email address _test@test.com_
2. Add event watch as per https://verbb.io/craft-plugins/formie/docs/developers/events#the-beforeSendNotification-event i.e.
```
use verbb\formie\events\SendNotificationEvent;
use verbb\formie\services\Submissions;
use yii\base\Event;

Event::on(Submissions::class, Submissions::EVENT_BEFORE_SEND_NOTIFICATION, function(SendNotificationEvent $event) {
    $event->notification->to = 'changedemail@test.com';
});
```
3. Set _Use Queue for Notifications_ to false
4. Submit the form
5. Check mail client (we use mailtrap for local dev) - you'll see  the 'to' address on the email is changedemail@test.com
6. Set _Use Queue for Notifications_ to false
7. Submit the form
8. Check mail client (we use mailtrap for local dev) - you'll see  the 'to' address on the default is test@test.com

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.2.22
- Craft version: Craft Solo 3.5.8
- Multi-site: No

**Additional context**
We've got the same form that needs to notify a different email address based on a number of factors (data specific and random); we don't want to duplicate the form as the data and integration will always be the same - it's just the notification 'to' and 'subject' that will change.

**Comment by engram-design** [comment]
Just fixed for the next release.

To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.23.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by anchovy** [comment]
Sweet, thanks @engram-design for the super-fast response!!

**Comment by engram-design** [comment]
Fixed in [1.2.24](https://github.com/verbb/formie/releases/tag/1.2.24)
